addons/stock/static/src/scss/report_stock_rule.scss

```diff
@@ -0,0 +1,97 @@
+.o_report_stock_rule_rule{
+    @include o-flex-display;
+    @include o-flex-flow(row, nowrap);
+}
+
+.o_report_stock_rule_putaway{
+    border-width: 2px;
+    border-style: solid;
+    border-color: black;
+    border-radius: 50%;
+    padding-left: 25px;
+    padding-right: 25px;
+    p{
+        text-align: center;
+        color: black;
+    }
+}
+
+.o_report_stock_rule_line{
+    @include o-flex(1,1,auto);
+    height: 20px;
+    line{
+        stroke: black;
+        stroke-width: 1;
+    }
+}
+
+.o_report_stock_rule_arrow{
+    @include o-flex(0,0,auto);
+    height: 20px;
+    width: 20px;
+    line{
+        stroke: black;
+        stroke-width: 1;
+    }
+    polygon{
+        fill: black;
+        fill-opacity: 0.5;
+        stroke: black;
+        stroke-width: 1;
+    }
+}
+
+.o_report_stock_rule_rule_name{
+    p{
+        text-align: center;
+    }
+}
+
+.o_report_stock_rule_legends{
+    @include o-flex-display;
+    @include o-flex-flow(row, wrap);
+    a{
+        display: block;
```

Comment:
is it necessary?

Reply:
needed if we want to put a background color or something else